src/components/attendee/MediaPlayer/MediaPlayer.tsx

```diff
@@ -0,0 +1,59 @@
+import { useCallback, useState } from "react";
+import {
+  faCompressArrowsAlt,
+  faExpandArrowsAlt,
+} from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import classNames from "classnames";
+
+import { IFRAME_ALLOW } from "settings";
+
+import { convertToEmbeddableUrl } from "utils/embeddableUrl";
+
+import styles from "./MediaPlayer.module.scss";
+
+interface MediaPlayerProps {
+  url: string;
+  autoPlay: boolean;
+}
+
+export const MediaPlayer: React.FC<MediaPlayerProps> = ({ url, autoPlay }) => {
+  const embedIframeUrl = convertToEmbeddableUrl({
+    url,
+    autoPlay,
+  });
+  const [expandedIframe, setExpandedIframe] = useState(false);
+
+  const toggleExpandedIframe = useCallback(() => {
+    setExpandedIframe((prevValue) => !prevValue);
+  }, [setExpandedIframe]);
+
+  const videoClassnames = classNames(styles.video, {
+    [styles.video__expanded]: expandedIframe,
+  });
+
+  return (
+    <div className={styles.componentMediaObject}>
```

Comment:
According to React, inline syntax is reinitialized on every re-render because there is no reference of a const. I'm not 100% sure how this works with classNames but we should look into in.

Reply:
@denisdimitrov as I understand it, everything is recalculated unless it is in `useMemo()` so a re-render does the same process be it `const classes = cn()` or `className={cn()}`, right?

Would be useful to have this info confirmed or disproved so that we'd have clearer code style recommendation.

Reply:
> so a re-render does the same recalculation be it const classes = cn() or className={cn()}, right?

Now that you mentioned it that way, it makes sense. I already read about the inline functions and looks like it was something that was a concern in Class components.

The only benefit of a `const` would be a reusability value and it's definition. Ex:

This will defined the class only once.
```tsx
const mainClass = cn(`flex centered`)

<div className={mainClass}>
  <div className={mainClass}>
  </div>
</div>
```

This will defined it twice:

```tsx
<div className={cn(`flex centered`)}>
  <div className={cn(`flex centered`)}>
  </div>
</div>
```